docs/docs/Releases/release200.md

```diff
@@ -0,0 +1,75 @@
+# Release 2.0.0
+
+Date: 1st Feb, 2023
+
+## Overview
+
+Klaw version 2.0.0 is a major release with several new features and
+improvements. Key highlights of this release include redesigned Request
+screens in the React UI, Schema promotion, codebase optimizations, and
+additional enhancements.
+
+## Breaking changes
+
+Java 11 Support discontinued: Klaw will no longer support Java 11.
+Instead, Klaw now supports Java 17 (LTS) and Java 19 (latest).
+
+## What's new in Klaw 2.0.0
+
+### Redesigned Request screens with React UI
+
+With the switch to React, this release features a redesign of the
+following user interfaces:
+
+-   **Topic Request**: Users can now submit topic requests using the new
+    UI.
+-   **Subscription Request** (Producer/Consumer): Users can now submit
+    ACL requests for both producers and consumers using the new UI.
+-   **Schema Request**: Users can now submit schema requests using the
+    new UI.
+
+![image](../../static/images/topic/RequestTopic-react.png)
+
+To preview the new Klaw user user interface, open the
+`application.properties` file on the Klaw **core** module, and set the
+value of the following property to `true`: :: \# Enable new Klaw user
```

Comment:
The following should be on a new line I missed this the first time I'm sorry, but the note looks perfect now.
# Enable new Klaw user interface 
klaw.coral.enabled=true

Reply:
This is the only one of the releases where it is still off